### PR TITLE
Coq of ocaml back

### PR DESCRIPTION
**Pull request by clarus**
For some reasons, `bubblewrap` prevented me to put the installation script in the build step.

### CODE REVIEW

**Comment by clarus** [comment]
I am attempting to add Ruby as a dependency for the tests (if this is not too heavy). Indeed, this is used for pre-processing during the compilation.

**Comment by palmskog** [comment]
@gares can you comment on this one? I'm personally not a fan of a package requiring more exotic system packages like Ruby without this being documented in the package itself.

File: released/packages/coq-of-ocaml/coq-of-ocaml.1.2.1/opam [hunk 1]
```diff
@@ -8,18 +8,16 @@ license: "MIT"
 build: [
   ["sh" "-c" "cd OCaml && ./configure.sh"]
   ["sh" "-c" "cd OCaml && make -j%{jobs}%"]
-  ["sh" "-c" "cd OCaml && make install"]
   [make "-j%{jobs}%"]
 ]
-remove: ["rm" "-R" "%{lib}%/coq/user-contrib/OCaml"]
+install: ["sh" "-c" "cd OCaml && make install"]
```
Comment (palmskog) [inline released/packages/coq-of-ocaml/coq-of-ocaml.1.2.1/opam]: Why not just: `install: [make "-C" "OCaml" "install"]`? Same for `build`.

**Comment by gares** [comment]
Example of a package needing ruby to build https://opam.ocaml.org/packages/yajl/yajl.0.7.0/ You could indeed specify the `{build}` attribute to the conf-ruby dep

**Comment by palmskog** [comment]
@clarus There should be no commas between `tags` strings. Please fix this so linting passes.

**Comment by clarus** [comment]
I amended the last commit.

**Comment by palmskog** [comment]
This looks good to me now, but I'll leave to @gares to definitely confirm and merge.

**Pull request merged**